core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,11 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+  .popover {
```

Comment:
@seanlip  Sure. Could you please look into #1774 ? I am confused with what to push into the CSS after discovering this PR with similar work